cpp/benchmarks/io/text/multibyte_split.cpp

```diff
@@ -77,30 +79,36 @@ static cudf::string_scalar create_random_input(int32_t num_chars,
   return cudf::string_scalar(std::move(*chars_buffer));
 }
 
-static void BM_multibyte_split(benchmark::State& state)
+static void bench_multibyte_split(nvbench::state& state)
 {
-  auto source_type      = static_cast<data_chunk_source_type>(state.range(0));
-  auto delim_size       = state.range(1);
-  auto delim_percent    = state.range(2);
-  auto file_size_approx = state.range(3);
+  auto const source_type      = static_cast<data_chunk_source_type>(state.get_int64("source_type"));
```

Comment:
```suggestion
  cudf::rmm_pool_raii pool_raii;
  
  auto const source_type      = static_cast<data_chunk_source_type>(state.get_int64("source_type"));
```
Similar to: https://github.com/rapidsai/cudf/blob/63a47d91fc558a5b0b89f3ce9b0a61ffb3ed84f5/cpp/benchmarks/stream_compaction/distinct.cpp#L29-L32
We need to include `#include <benchmarks/fixture/rmm_pool_raii.hpp>` and initialize the pool first before doing any (de)allocations.

Reply:
done, thanks!